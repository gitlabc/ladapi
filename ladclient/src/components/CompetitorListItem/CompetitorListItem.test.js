import React from 'react';
import renderer from 'react-test-renderer';
import CompetitroListItem from './CompetitorListItem';
import {shallow} from 'enzyme';

describe('CompetitroListItem', () => {
    
    
    test('should be passed', () => {
        expect(true).toBe(true);
    });
    
    // test('should toggles erro type when show/hide password clicked', () => {
    //     const wrapper1 = shallow(<CompetitroListItem 
    //         htmlId="example-optional"
    //         label="First Name"
    //         name="firstname"
    //         onChange={() => { }}
    //         required
    //     />);

    //     expect(wrapper1.find('div')).toHaveLength(1);
        
    //     const wrapper2 = shallow(<CompetitroListItem 
    //         htmlId="example-optional"
    //         label="First Name"
    //         name="firstname"
    //         onChange={() => { }}
    //         required
    //         error="First name is required."            
    //     />);

    //     expect(wrapper2.find('div')).toHaveLength(2);
    // });
    

    // test('should hides error by default', () => {
    //     const tree = renderer.create(
    //         <CompetitroListItem
    //             htmlId="example-optional"
    //             label="First Name"
    //             name="firstname"
    //             onChange={() => { }}
    //             required
    //         />
    //     ).toJSON();

    //     expect(tree).toMatchSnapshot();
    // });

    // test('should shows error when error is set', () => {
    //     const tree = renderer.create(
    //         <CompetitroListItem
    //             htmlId="example-optional"
    //             label="First Name"
    //             name="firstname"
    //             onChange={() => { }}
    //             required
    //             error="First name is required."
    //         />
    //     ).toJSON(); 

    //     expect(tree).toMatchSnapshot();
    // });
    
});
