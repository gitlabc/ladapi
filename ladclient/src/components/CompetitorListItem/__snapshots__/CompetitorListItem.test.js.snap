// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompetitroListItem should generate the CompetitorListItem 1`] = `
<div
  className="ant-row-flex ant-row-flex-space-around ant-row-flex-middle"
  style={undefined}
>
  <div
    className="ant-col-xs-1 ant-col-sm-1"
  >
    <div
      className=""
    />
     
  </div>
  <div
    className="ant-col-xs-8 ant-col-sm-9"
  >
    <div
      className={undefined}
    >
      1
      .
      PROF. RAYMOND TREMBLAY III
      (
      2
      )
    </div>
  </div>
  <div
    className="ant-col-xs-8 ant-col-sm-9"
  >
    2.2,2.4,3.6,5.7,6.6,8.2,9.2,9.4
  </div>
  <div
    className="ant-col-xs-2 ant-col-sm-1"
  >
    <button
      className="ant-btn ant-btn-primary ant-btn-sm"
      onClick={[Function]}
      onMouseUp={[Function]}
      type="button"
    >
      <span>
        12.23
      </span>
    </button>
  </div>
  <div
    className="ant-col-xs-2 ant-col-sm-1"
  >
    <button
      className="ant-btn ant-btn-primary ant-btn-sm"
      onClick={[Function]}
      onMouseUp={[Function]}
      type="button"
    >
      <span>
        30.86
      </span>
      <span>
         
      </span>
    </button>
  </div>
</div>
`;
